src/test/java/com/google/javascript/clutz/enum.d.ts

```diff
@@ -1,3 +1,12 @@
+declare namespace ಠ_ಠ.clutz.some {
+  enum EscapedEnum {
+    A = '\\' ,
```

Comment:
are you dealing with the non-identifier enum key in a different CL?

Reply:
Yes the other fix was submitted in https://github.com/angular/clutz/pull/768